packages/front-end/components/Radix/Button.tsx

```diff
@@ -17,7 +17,7 @@ export type Props = {
   setError?: (error: string | null) => void;
   icon?: ReactNode;
   iconPosition?: "left" | "right";
-  children: string | string[];
+  children: ReactNode;
```

Comment:
Updated -- I am concerned that in the future we will have a Button component with 30 props because we will need to surface all underlying children props to the consumer, but we are not there yet so I rolled this back and added `textWeight` as a new prop.

Reply:
I agree.  Looking at the figma, it looks like there's a mismatch between our design system and Radix.  Radix uses bold for all buttons except for ghost variants.  However, in our design system the ghost buttons also use bold.

So instead of a prop, we should either fix our design system to match Radix or add some custom CSS within the Button component to force ghost buttons to be bold.

Reply:
I have reverted the changes and will keep all buttons with weight `medium` for now.

If we feel more strongly in the future that all ghost buttons should be `600` then we can easily add that rule -- but will default to keep all buttons the same at the moment.